lms/static/js/student_account/views/RegisterView.js

```diff
@@ -88,6 +89,13 @@
                         }
                     )
                 );
+
+                // Added by Labster
+                // Hide one of the ip address error message to avoid a duplication message.
+                if ('<li>' + this.ipAddressWarningMsg + '</li>' == this.errors){
+                    $(this.el).find('.submission-warning').hide();
+                }
```

Comment:
@lukyfebrianto why do we have a duplication?

Reply:
@polesye , because there are 2 message in registration for IP address message
1. message for the first view, https://github.com/Livit/Livit.Learn.EdX/pull/104/files#diff-558beedb32b9d68d46c2e88a3baa44ecR1
2. message after request post form validation, https://github.com/Livit/Livit.Learn.EdX/pull/104/files#diff-55b798ee23a7fde8d1103408afcd0f16R1641

Why I create new html for warning message instead of using the old one because I think it's more complicated(we also need create/changes some js to handle it).

Reply:
> Why I create new html for warning message instead of using the old one because I think it's more complicated(we also need create/changes some js to handle it).

@lukyfebrianto Could you elaborate? Why is it more complicated if according to the code above it doesn't require any changes from us to make it shown.

Reply:
So @lukyfebrianto, what's wrong with this view https://user-images.githubusercontent.com/25189966/35797941-372b2558-0a59-11e8-85ea-305ce4eea66a.png
![image](https://user-images.githubusercontent.com/1247491/35809504-8bb0a700-0a91-11e8-94a5-aa9f22bd2494.png)
 ? 
Do we have any requirements to show it outside the form ?

Can we always show it this way and leave the code as simple as possblie?